hadoop-hdds/container-service/pom.xml

```diff
@@ -213,6 +213,13 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.ozone</groupId>
+      <artifactId>hdds-server-framework</artifactId>
+      <version>${hdds.version}</version>
+      <type>test-jar</type>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Please add this without `scope` in `hadoop-hdds/pom.xml`:

```xml
    <dependency>
      <groupId>org.apache.ozone</groupId>
      <artifactId>hdds-server-framework</artifactId>
      <version>${hdds.version}</version>
      <type>test-jar</type>
    </dependency>
```

after:

https://github.com/apache/ozone/blob/9d6b6928a7437d344a37b74c426e1227d24d1f24/hadoop-hdds/pom.xml#L143-L147

and omit `version` here in `hadoop-hdds/container-service/pom.xml` and in `hadoop-hdds/server-scm/pom.xml`.

(See `hdds-common` as an example.)